### ISSUE TITLE
Не работает подавление ошибки //@skip-check using-isinrole

### ISSUE THREAD

**Issue by DmitryShvaika** [1C-Company/1c-edt-issues#409]
### Описание ошибки

Не работает подавление ошибки //@skip-check using-isinrole

### Как воспроизвести

1. Введите текст ниже в серверный модуль

```
Функция РольДоступнаПользователю(ИмяРоли) Экспорт
	
	//@skip-check using-isinrole
	Возврат РольДоступна(ИмяРоли);
	
КонецФункции // РольДоступнаПользователю

```
2. Наблюдайте ошибку

### Скриншоты

![image](https://user-images.githubusercontent.com/54451084/160235601-e9809c27-500a-4fc2-a200-2cababaaf618.png)

### Ожидаемое поведение

Подавление ошибки должно работать

### Лог рабочей области

[логи.zip](https://github.com/1C-Company/1c-edt-issues/files/8355175/default.zip)

### Версия 1С:EDT

Ruby 2021.3.3

### Операционная система

Windows

**Comment by vadimeg** [comment]
В текущей версии не воспроизводится. Должно работать в ближайшем исправительном релизе. 2021.3.5

**Comment by DmitryShvaika** [comment]
Да, на тестовой конфигурации то же не получилось воспроизвести. Видимо влияет текст выше или ниже функции. Закрываю задачу.
